src/components/application_manager/src/resumption/resumption_data_processor.cc

```diff
@@ -640,9 +649,12 @@ void ResumptionDataProcessor::AddPluginsSubscriptions(
   LOG4CXX_AUTO_TRACE(logger_);
 
   for (auto& extension : application->Extensions()) {
+    LOG4CXX_DEBUG(logger_, "Entered subscriber");
     extension->ProcessResumption(
         saved_app,
         [this](const int32_t app_id, const ResumptionRequest request) {
+          LOG4CXX_AUTO_TRACE(logger_);
```

Comment:
@mked-luxoft Do we trace lambda? May be `LOG4CXX_DEBUG` on line 657 will be enough?

Reply:
@ZhdanovP removed in 23e9b67